lib/gamestate.rb

```diff
@@ -0,0 +1,120 @@
+require 'json'
+require_relative 'player'
+require_relative 'room'
```

Comment:
castle.rb requires room.rb, so if you require the latter you needn't require the former. Applies elsewhere too.